docs/deployment/getting_started_locally.md

```diff
@@ -230,10 +230,18 @@ cat <<EOF | sudo tee -a /etc/hosts
 127.0.0.1 api.e2e-upd-node.local.internal.local.gardener.cloud
 127.0.0.1 api.e2e-upd-node-wl.local.external.local.gardener.cloud
 127.0.0.1 api.e2e-upd-node-wl.local.internal.local.gardener.cloud
+127.0.0.1 api.e2e-upd-zone.local.external.local.gardener.cloud
+127.0.0.1 api.e2e-upd-zone.local.internal.local.gardener.cloud
+127.0.0.1 api.e2e-upd-zone-wl.local.external.local.gardener.cloud
+127.0.0.1 api.e2e-upd-zone-wl.local.internal.local.gardener.cloud
```

Comment:
These were left out intentionally, see
https://github.com/gardener/gardener/blob/141ccd139c7b0bc717b381a7d3884db8bdfa64cb/hack/test-e2e-local.sh#L72-L78

Reply:
The problem is, that these lines complain about the missing host entries:
https://github.com/gardener/gardener/blob/141ccd139c7b0bc717b381a7d3884db8bdfa64cb/hack/test-e2e-local.sh#L92-L95

I.e., I simply added the lines to `/etc/hosts` to silence the warning.

Reply:
Fair point, I opened https://github.com/gardener/gardener/pull/8040

Reply:
Thanks. I dropped 99c8053 now that #8040 is merged.